### PR TITLE
[15.0][MIG] purchase_stock_picking_return_invoicing

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
This module extends the functionality of purchase orders to better manage
supplier returns and refunds.

In the purchase order you are able to display, for each line:

* Billed Quantity and Refunded Quantity, as separate fields.

* Received Quantity and Returned Refundable Quantity, as separate fields.

You have the option to create a vendor bill or a refund. In the bill or refund
the correct quantity will be proposed, based on those fields.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
please rebase

**Review by MiquelRForgeFlow — APPROVED** [review]

File: purchase_stock_picking_return_invoicing/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# Copyright 2017-2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Stock Picking Return Invoicing",
+    "summary": "Add an option to refund returned pickings",
+    "version": "15.0.1.2.0",
```
Comment (sergio-teruel) [inline purchase_stock_picking_return_invoicing/__manifest__.py]: Change by 15.0.1.0.0

**Review by sergio-teruel — APPROVED** [review]
Minor comment

File: purchase_stock_picking_return_invoicing/__manifest__.py (hunk 1 above)
Comment (JuanyDForgeflow) [reply purchase_stock_picking_return_invoicing/__manifest__.py]: @sergio-teruel it's ok now thanks for letting me know

Comment (sergio-teruel) [reply purchase_stock_picking_return_invoicing/__manifest__.py]: Thanks @JuanyDForgeflow

**Pull request merged**